components/counter.js

```diff
@@ -4,12 +4,43 @@ const Counter = () => {
   const [tenthSeconds, setTenthSeconds] = useState(0);
 
   useEffect(() => {
-    const intervalId = setInterval(() => {
+    let intervalId;
+    let hidden = "";
+    let visibilityChange = "";
+
+    if (typeof document.hidden !== "undefined") {
```

Comment:
It looks like `document.hidden` has pretty wide browser support: https://developer.mozilla.org/en-US/docs/Web/API/Document/hidden#browser_compatibility

You might not need all these conditionals.

Reply:
Thank you for the information! , it looks like the conditionals may not be necessary. I will review the code accordingly and consider removing the unnecessary conditionals